whois-query/src/test/java/net/ripe/db/whois/query/support/QueryExecutor.java

```diff
@@ -31,12 +30,10 @@
     private static final Set<String> ERRORS_TO_IGNORE = Sets.newHashSet("ERROR:101:", "ERROR:106:");
 
     private final QueryExecutorConfiguration configuration;
-    private final AccessControlListManager accessControlListManager;
     private final Logger logger;
 
-    public QueryExecutor(final QueryExecutorConfiguration configuration, final AccessControlListManager accessControlListManager, final Logger logger) throws UnknownHostException {
```

Comment:
Confirmed aclManager is not needed in test support class - personal object accounting is done with hardcoded object type check.